vms/vm-under-test/scripts/first-boot

```diff
@@ -22,6 +22,8 @@ set -e
 mkdir /mnt/huge
 mount /mnt/huge --source nodev -t hugetlbfs -o pagesize=1GB
 
+systemctl mask "$(systemctl --type swap | grep '.swap' | awk '{print $1}')"
```

Comment:
Is this command needed if the swap is removed from `/etc/fstab`?

Reply:
yes